files/build_templates/docker_image_ctl.j2

```diff
@@ -151,11 +151,11 @@ function waitForAllInstanceDatabaseConfigJsonFilesReady()
                     fi
 		done
             done
-	fi
+        fi
+        # delay a second to allow the file to be fully accessible
```

Comment:
@mlok-nokia could you add a comment here as discussed on to why we need this delay. This delay is needed to make sure that the database_config.json files are correctly rendered from j2 template files ( renderning takes some time )

Reply:
> @mlok-nokia could you add a comment here as discussed on to why we need this delay. This delay is needed to make sure that the database_config.json files are correctly rendered from j2 template files ( renderning takes some time )

@judyjoseph  I have updated it with comment which you mentioned above.